src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(" Figure: circle, area:" + getArea() + "sq. units, side: "
+                + radius + "units, color: " + getColor());
```

Comment:
In the draw method, the term 'side' is used, which is incorrect for a circle. It should be 'radius' to accurately describe the property of the circle.